runtime/cmake/caches/osx_x86-64_cmprssptrs.cmake

```diff
@@ -1,5 +1,5 @@
 ################################################################################
-# Copyright (c) 2018, 2019, 2019 IBM Corp. and others
+# Copyright (c) 2018, 2020 IBM Corp. and others
```

Comment:
copyright bumped here but no changes made to file

Reply:
thought I'd fix the copyright typo (duplication of `2019`), but I can remove this change and assume it will be fixed the next time someone changes this file?

Reply:
I would leave the change in.